src/main/webapp/index-lti.vm

```diff
@@ -108,3 +115,18 @@
   </body>
 
 </html>
+#else
+  <html>
+    <head>
+      <meta charset="utf-8">
+      <link rel="stylesheet" href="vendor/bootstrap/bootstrap.min.css">
+      <title>Nor allowed</title>
+    </head>
+    <body>
+      <br><br>
+      <div class="alert alert-info">
+        Sorry - this functionality is only for instructors.
```

Comment:
I think more generic message world be Not allowed, instead of telling who will get access to the tool